exception TestError();

class TestClass() {
	~ {
		throw TestError();
	}
}

